src/toolchain/export-web.md

```diff
@@ -0,0 +1,157 @@
+<!--
+  ~ This Source Code Form is subject to the terms of the Mozilla Public
+  ~ License, v. 2.0. If a copy of the MPL was not distributed with this
+  ~ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+-->
+
+
+# Intro to Web Builds
+
+Web builds are a fair bit more difficult to get started with compared to native builds.
+This will be a complete guide on how to get things compiled.
+However, setting up a web server to host and share your game is considered out of scope of this guide, and is best explained elsewhere.
+
+```admonish warning
+Web support with gdext is experimental and should be understood as such before proceeding.
+```
+
+
+## Installing Pre-Requisites
+
+Install a nightly build of `rustc`, the `wasm32-unknown-emscripten` target for `rustc`, and `rust-src`.
+The reason why nightly `rustc` is required is the unstable flag to build `std` ([`-Zbuild-std`][flag-build-std]).
+Assuming that Rust was installed with `rustup`, this is quite simple.
+
+[flag-build-std]: https://doc.rust-lang.org/cargo/reference/unstable.html#list-of-unstable-features
+
+  ```sh
+  rustup toolchain install nightly
+  rustup component add rust-src --toolchain nightly
+  rustup target add wasm32-unknown-emscripten --toolchain nightly
+  ```
+
+Next, install Emscripten.  The simplest way to achieve this is to install [`emsdk` from the git repo][emsdk-git].
+We recommended version 3.1.39 for now.[^1]
+
+[emsdk-git]: https://github.com/emscripten-core/emsdk#readme
+
+```sh
+git clone https://github.com/emscripten-core/emsdk.git
+cd emsdk
+./emsdk install 3.1.39
+./emsdk activate 3.1.39
+source ./emsdk.sh     (or ./emsdk.bat on windows)
+```
+
+It would also be **highly** recommended to follow the instructions in the terminal to add `emcc`[^2] to your `PATH`.
+If not, it is necessary to manually `source` the `emsdk.sh` file in every new terminal prior to compilation.
+This is platform-specific.
+
+
+## Project Configuration
+
+Enable the `experimental-wasm` feature on gdext in the `Cargo.toml` file.
+It is also strongly recommended to enable the `lazy-function-tables` feature to avoid an extremely long compile time with release builds.
+It is likely to due a bug, but is still being investigated.  Edit the line to something like the following:
+
+```toml
+godot = { git = "https://github.com/godot-rust/gdext", branch = "master", features = ["experimental-wasm", "lazy-function-tables"] }
+```
+
+If you do not already have `.cargo/config.toml` file, do the following:
+
+- Create a `.cargo` directory at the same level as your `Cargo.toml`.
+- Inside of that directory, create a `config.toml` file.
+
+This file needs to contain the following:
+
+```toml
+[target.wasm32-unknown-emscripten]
+rustflags = [
+    "-C", "link-args=-sSIDE_MODULE=2",
+    "-C", "link-args=-pthread",
```

Comment:
Thanks for changing this!
Sorry I wasn't very clear before, maybe I'd still mention the old one just in case someone is used to that (otherwise they might wonder where it is).

That is:
```suggestion
    "-C", "link-args=-pthread", # was -sUSE_PTHREADS=1 in earlier versions
```

Reply:
What part not to mention? Sorry. If you mean line 112 then... that's kinda confusing. I can remove the images about the error for missing templates, I guess.  They don't really add much besides push the page length; it feels like anyway.

Reply:
Just apply my suggestion here, so that people familiar with the old flag aren't confused 🙂 

> I can remove the images about the error for missing templates, I guess. They don't really add much besides push the page length; it feels like anyway.

Yeah, would be good to focus more on the WASM topic, you can remove them.